jenkins/scripts/integration_clean.sh

```diff
@@ -22,10 +22,17 @@ function cleanup() {
     startswith("ci-test-vm-")) | select((.Name | ltrimstr("ci-test-vm-") |
     split("-") | .[0] | strptime("%Y%m%d%H%M%S") | mktime) < (now - 21600))
     | .ID ')
+
+  DEBUGGING_VM_LIST=$(openstack server list -f json | jq -r '.[] | select(.Name |
+  startswith("ci-test-vm-")) | select((.Name | ltrimstr("ci-test-keep-vm-") |
```

Comment:
I expect a VM named 'ci-test-keep-vm-' won't match the selection of `startswith("ci-test-vm-")`.